### PR TITLE
Create DELETE API handler for Todo list (#41)

### PR DESCRIPTION
**Pull request by LincRui**
**Issue:**
#41 

**Solution:**
 Created a DELETE API endpoint that receives an entry id and removes the corresponding entry from the database

**Test Method:**
Manually test with Postman on localhost:9000/todo.
Unit testing with the mock firebase and database using Jest and SuperTest.

**Risk:**
No data validation.

**Reviewed By:**
Raymond Chiu, Jonathan Thamrin

### CODE REVIEW

**Review by Pranavan10 — APPROVED** [review]

File: server/routes/tests/delete.todo.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const app = require("../../app");
+
+const expectedAllTodo = [
+    0: {
```
Comment (jonathan-thamrin) [inline server/routes/tests/delete.todo.test.js:5]: There seems to be syntax errors with how this array is set up. Can be fixed with the following :)

```suggestion
    {
```

File: server/routes/tests/delete.todo.test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        date: "21/03/2021",
+        description: "sleep",
+        entry_id: 1,
+        ticked: false,
+        time: "10:00",
+        title: "Shopping",
+    },
+    1: {
```
Comment (jonathan-thamrin) [inline server/routes/tests/delete.todo.test.js:13]: Refer to previous comment for reason.
```suggestion
    {
```

File: server/routes/tests/delete.todo.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        description: "Go shopping with bob",
+        entry_id: 2,
+        ticked: false,
+        time: "08:00",
+        title: "Shopping",
+    },
+];
+
+const expectedAfterDeleteTodo = [
+    0: {
```
Comment (jonathan-thamrin) [inline server/routes/tests/delete.todo.test.js:24]: Refer to previous comment for reason.
```suggestion
    {
```

**Review by jonathan-thamrin — CHANGES_REQUESTED** [review]
Small nitpick but don't forget to apply formatting with Prettier

File: server/routes/tests/delete.todo.test.js (hunk 3 above)
Comment (LincRui) [reply server/routes/tests/delete.todo.test.js:24]: Thanks, bro. Learned it.

**Comment by raymondhonsumchiu** [comment]
Might need to apply recent changes in main via rebasing before you can merge this pull request

**Review by raymondhonsumchiu — APPROVED** [review]
Everything looks good now!

**Review by jonathan-thamrin — APPROVED** [review]

**Pull request merged**